app/models/notification.rb

```diff
@@ -13,38 +15,9 @@
 class Notification < ApplicationRecord
   belongs_to :user
 
-  after_save :send_gcm_notification
-
   scope :unread, -> { where read: false }
 
   def read!
     update!(read: true)
   end
-
-  def create
-
-  end
-
-  def send_gcm_notification
-    return if read
-
-    begin
-      if !Rpush::Gcm::App.find_by_name("tappb")
-        app = Rpush::Gcm::App.new
-        app.name = "tappb"
-        app.auth_key = Rails.application.secrets.fcm_secret
-        app.connections = 1
-        app.save!
-      end
-
-      n = Rpush::Gcm::Notification.new
-      n.app = Rpush::Gcm::App.find_by_name("tappb")
-      n.registration_ids = user.android_device_registration_tokens.all.map { |r| r.token }
-      n.data = {body: message, title: "Tabbp notification"}
-      n.priority = 'high'
-      n.content_available = true
-      n.save!
-    rescue
-    end
-  end
```

Comment:
Dit zijn denk ik geen rubocop changes?

Reply:
Dat klopt (voor het tweede deel, de `create` wordt nergens gebruikt). Al gaat dit deel verwijderd worden in een andere PR. Ik zal de changes terugzetten, en rubocop disablen voor die methode.